test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class UserTest < ActiveSupport::TestCase
+  # test "the truth" do
```

Comment:
@amitshethe This file can be deleted

Reply:
corrected them in controller setup branch